data/css/escola.scss

```diff
@@ -191,10 +237,15 @@ where it needs to be */
       font-size: 16px;
       -EknFormattableLabel-text-transform: 'uppercase';
       font-weight: normal;
-      color: #b3b3b3;
+      color: $card-context-color;
     }
     &__thumbnail {
-      background-color: $base-color;
+      background-color: $background-dark-color;
+    }
+    &.width-d.height-b &__contentFrame,
+    &.width-e.height-b &__contentFrame,
+    &.width-e.height-c &__contentFrame {
```

Comment:
Watch out with the `&` here! `&` means "everything that comes higher in the selector hierarchy", so what you get from:

```
 &.width-e.height-c &__contentFrame {
``` 

is actually:

```
.ArrangementGrid .CardThumb.width-e.height-c .ArrangementGrid .CardThumb__contentFrame {
      padding: 20px;
}
```

Notice the `.ArrangementGrid` in the middle, therefore this selector won't work.